src/content/overview/fleet-management/cluster-management/installation-name/index.md

```diff
@@ -0,0 +1,42 @@
+---
+linkTitle: Installation names
+title: Picking an installation name
+description: Every Giant Swarm installation has a unique name. Learn the rules to select a proper name for a new installation.
+weight: 100
+menu:
+  principal:
+    parent: overview-fleet-management-clusters
+    identifier: overview-fleet-management-clusters-installation-name
+user_questions:
+  - How do I determine the name for my Giant Swarm installation / management cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2024-11-28
+---
+
+A Giant Swarm installation is the management cluster where all our platform components run. It's attached to a cloud provider account and region, or an on-premise data center, and it's the central point and API to use the platform capabilities.
+
+## Why an installation name
+
+Every Giant Swarm installation has an unique name. This name helps to identify the installation in our systems and in your communication with Giant Swarm.
+
+When setting up a new installation for you, a name is needed. Think of this name as your handy alias for everything you do with Giant Swarm in a specific infrastructure provider region, for example _AWS Frankfurt (eu-central-1)_.
+
+Once a name is decided on, you will use it often in your communication with Giant Swarm. Of course, you can also use it in your internal communication.
+
+## Criteria for good installation names
+
+There are many criteria which distinguish a good installation name. It should be easy to pronounce in an English language context, easy to type, and ideally easy to memorize. On the other hand, it shouldn't reveal information about you as a customer.
```

Comment:
Where did the hint about DNS records go? That's important as the reason for this sentence.

Reply:
hmm Jose wrote before we dont use installation name anymore in the DNS